buildSrc/src/main/groovy/MuzzlePlugin.groovy

```diff
@@ -358,6 +358,9 @@ class MuzzlePlugin implements Plugin<Project> {
       config.dependencies.add(dep)
     }
     for (String additionalDependency : muzzleDirective.additionalDependencies) {
+      if (additionalDependency.count(":") < 2) {
```

Comment:
What does this condition mean?

Reply:
Added comment, thanks

Reply:
Use the same version as the main artifact? Maybe we should use some placeholder like `$MAIN_LIB_VERSION` and stirng.replace() it here.

Reply:
I thought of a placeholder approach - I couldn't use `$` though since then users have to be careful on single quotes vs double quotes or the variable will be replaced by Groovy, not us. Just checking for a versionless artifact seems reasonably simple to me (or I could use a different template syntax like `|VERSION|`